Permanent/Models/ChangePasswordResponse.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  ChangePasswordResponse.swift
+//  Permanent
+//
+//  Created by Lucian Cerbu on 14.01.2021.
+//  Copyright © 2021 Victory Square Partners. All rights reserved.
+//
+
+import Foundation
+// MARK: -PasswordChange Response
+struct ChangePasswordResponse: Codable {
```

Comment:
Please use the "Model" protocol here. See how **RecordVO** is implemented (and **APIResults** class).

Reply:
Used APIResults class for API response.